### ISSUE TITLE
False positive: '--' marker rule triggers inside source code blocks

### ISSUE THREAD

**Issue by raifdmueller** [docToolchain/asciidoc-linter#54]
## Description

The list marker rule ("Missing space after the marker '--'") triggers on CLI options like `--kunde`, `--adresse` etc. inside `[source,bash]` code blocks.

## Steps to Reproduce

Given this AsciiDoc file:

```asciidoc
[source,bash]
----
lorem ipsum
----
```

Running `asciidoc-linter file.adoc` reports:

```
✗ file.adoc, line 3: Missing space after the marker '--'
✗ file.adoc, line 4: Missing space after the marker '--'
✗ file.adoc, line 5: Missing space after the marker '--'
✗ file.adoc, line 6: Missing space after the marker '--'
✗ file.adoc, line 7: Missing space after the marker '--'
```

## Expected Behavior

Content inside `----` delimited blocks should be skipped by all rules, since it is literal/source content, not actual AsciiDoc structure. The `--` in `--kunde` is a CLI option prefix, not an AsciiDoc list marker.

## Actual Behavior

The linter interprets `--kunde` as a malformed AsciiDoc `--` list marker missing a space.

## Environment

- asciidoc-linter 0.1.0 (installed from git main branch)

## Note

This is related to #52 (same root cause: rules not skipping code block content).

**Comment by rdmueller** [comment]
## Analysis

Already fixed on `main` by #56 (commit 2ded7ad). `WhitespaceRule.check()` skips every line returned by `find_code_block_content_lines()`, so `--kunde` inside a `----` block no longer counts as a list marker.

Verified: a `[source,bash]` listing with `--kunde`, `--adresse` and `--x` lines produces no WS001 finding.

The same reproduction, placed at the very start of a file, still reports BLOCK002 "Block should be preceded by a blank line". That's a different bug and is tracked in #64.

Suggestion: close as completed.

---
_Generated by [Claude Code](https://claude.ai/code)_
